debian/install

```diff
@@ -1,2 +1,3 @@
 debian/homeserver.yaml etc/matrix-synapse
 debian/log.yaml etc/matrix-synapse
+debian/matrix-synapse.service lib/systemd/system
```

Comment:
> fwiw we can do it the "right way" when we rename the package, I didn't find a way for dh_systemd to install it under a name that wasn't the package name. :/

I'm totally failing to follow your logic here. We aren't using dh_systemd* at all currently (because we are on debhelper compat 9, which doesn't enable it by default, and instead dh_installinit`does the installation of the service file).

Either way, the solution is just to add a `--name` argument in the relevant `override_dh_*` sections.